docs/ecosystem-eslint-plugin-testing-library.md

```diff
@@ -0,0 +1,23 @@
+---
+id: ecosystem-eslint-plugin-testing-library
+title: eslint-plugin-testing-library
+---
+
+[`eslint-plugin-testing-library`][gh] is an ESLint plugin for `Testing Library`
+that helps users to follow best practices and anticipate common mistakes when
+writing tests.
+
+![eslint-plugin-testing-library example showing an error](https://i.imgur.com/UKr9jVk.png)
```

Comment:
Could we store this image within the repository rather than on imgur? I believe some countries/companies block imgur.

Reply:
sure! I did that at first but wasn't sure if it was the right decision. Included at the repo now.